### ISSUE TITLE
Tag and release project

### ISSUE THREAD

**Issue by markhobson** [govuk-one-login/service-header#46]
Although there is a [CHANGELOG](https://github.com/govuk-one-login/service-header/blob/main/CHANGELOG.md) for this project, there are no milestones, tags or releases.

It would be useful for downstream projects if there was a Git tag for each release, and ideally a [GitHub release](https://github.com/govuk-one-login/service-header/releases) too.

**Comment by danacotoran** [comment]
Hi @markhobson, thanks for your suggestions!

I'd like to understand more about the way you use the code in this repo:
As you know this service header is not currently distributed on any package hosting service. But would it be useful to you if it was?
Even though we are not releasing and distributing this project via e.g. `npm`, would it still be helpful to you if Git tags and releases were used?

Many thanks! 🙌

**Comment by markhobson** [comment]
Hi @danacotoran, thanks for your response.

We're currently using the service header in a Python Flask app. This requires us to provide the template using Jinja instead of Nunjucks (see #25). Ideally this would be provided via a pip package, but in the meantime we copy & paste the content from the Nunjucks template at a specific commit.

We also need to build the web assets for the service header. Ideally this would be provided via a npm package like [govuk-frontend](https://frontend.design-system.service.gov.uk/installing-with-npm/) does, but for now we add this repo as a [GitHub URL](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#github-urls) at a specific commit. We then build the assets from source since we [customise the settings](https://frontend.design-system.service.gov.uk/sass-api-reference/).

Both these tasks require a specific commit to keep them aligned. This is where a Git tag would be helpful - we could obtain the template for that tag and add the npm package at that tag. We currently have to use a recent Git commit hash. This also requires us to guess which commit matches which release in the CHANGELOG, which a Git tag would resolve.

So in summary:

1. A Git tag for each release would be a better than a commit for our current process
2. A npm package would improve the process to build our web assets (and presumably help [Nunjucks-based projects](https://frontend.design-system.service.gov.uk/use-nunjucks/) consume the template)
3. A Jinja template and pip package would improve how we consume the template (covered by #25)

**Comment by markhobson** [comment]
Any chance of creating a git tag when you [release a version](https://github.com/govuk-one-login/service-header/commit/3cada5ea7556cc82656dc81d8c01cbffa775e74a) please?

**Comment by danacotoran** [comment]
> unless there are also plans to publish npm packages

We haven't made it very obvious, but the service header can already be installed with `npm` CLI direct from Github. 
No plans to publish to `npm` though it was on the table initially – we just hit a wall in terms of organisation/ownership stuff (for obvious reasons it should not be published under someone's personal account), and other priorities took centre stage.
